status_cake_exporter/app.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import os
+import sys
+import time
+
+import typer
+from prometheus_client import REGISTRY, start_http_server
+
+from ._logging import configure_logging
+from ._test_collector import TestCollector
+
+logger = logging.getLogger("exporter")
+
+
+def exporter(
+    username: str = typer.Option(
+        "",
+        help="Username for the account. This is only required for legacy accounts. (DEPRECATED)",
+        envvar="USERNAME",
+    ),
```

Comment:
Can this be removed now if legacy API endpoints are no longer being called?

Reply:
Possibly yeah. I marked it as deprecated because I wasn't sure if it would have an impact on how users were consuming the app.